## 1. Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде. 
### Результаты: 
1) Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550.<br /> 
2) Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228.<br />
3) Конверсии : conv1 = 1.05% conv2 = 1.47%.<br />
## 2. Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?

## Решение:
Для оценки статистической значимости различий между конверсиями в двух группах при α=5% можно использовать Z-критерий для разности долей. Формула Z-критерия:

Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))

где p = (n1 * p1 + n2 * p2) / (n1 + n2)

p1 и p2 - конверсии в группах, N1 и N2 - количество юзеров в группах, которые заходили на сайт в период эксперимента, n1 и n2 - количество юзеров, которые совершили хотя бы одну покупку за период эксперимента.

In [1]:
from math import sqrt

p1 = 164 / 15550

p2 = 228 / 15550

n1 = 164

n2 = 228

N1 = 15550

N2 = N1

p =  (n1 * p1 + n2 * p2) / (n1 + n2)

Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))
Z

-3.2110930329358367

## Ответ:
### 1. Критическое значение Z-критерия при α=5% и двухсторонней альтернативе = 1.96. Так как расчетное значение Z меньше критического значения (-3.21 < 1.96) => нулевая гипотеза о равенстве конверсий в группах отвергается. То есть, изменения внесенные в заказную форму в магазине Утконос дали статистически значимый эффект на уровень конверсии в покупку. Рекомендуется использовать новую форму заказа.
### 2. Если размер выборки достаточно большой (более 30 элементов в каждой группе), то может быть использован Z-критерий. В данном случае размер выборки составляет 10 000 элементов, следовательно нужно использовать Z-критерий.